yt_dlp/extractor/moviepilot.py

```diff
@@ -0,0 +1,121 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import (
+    int_or_none,
+    parse_duration,
+    parse_iso8601,
+    unescapeHTML,
+)
+
+
+class MoviepilotIE(InfoExtractor):
+    _IE_NAME = "moviepilot"
+    _IE_DESC = "Moviepilot trailer"
+    _VALID_URL = r'https?://(?:www\.)?moviepilot\.de/movies/(?P<id>[^/]+)/?.*'
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?moviepilot\.de/movies/(?P<id>[^/]+)'
```
it doesnt do anything